tests/unit/MigrationTest.js

```diff
@@ -639,4 +640,72 @@ describe('Migrations', () => {
                     });
                 }));
     });
+
+    describe('CheckPreviousReportActionID', () => {
+        // Note: this test has to come before the others in this suite because Onyx.clear leaves traces and keys with null values aren't cleared out between tests
```

Comment:
Why is this? I tried moving the tests and there was no change and all tests passed.

Reply:
Hi I saw this note in a previous test [here](https://github.com/Expensify/App/blob/975146fa30bc65a19df8e0cca525af32ce2ceda5/tests/unit/MigrationTest.js#L153). Reordering was also passing for me, but I was not sure if this was an edge case in onyx and doesn't happen everytime. So added for clarity. Should I remove it?

Reply:
Can we remove this? Even if we change the order there is no change in result?

Reply:
done